index.js

```diff
@@ -32,7 +33,7 @@ CacheBreaker.prototype.cacheBreakPath = function(base, resource) {
   var extname = path.extname(resource);
   var basename = path.basename(resource, extname);
 
-  return path.join(dirname, basename + '.' + cs.substring(0, 10) + extname);
+  return url.resolve(dirname + '/', basename + '.' + cs.substring(0, 10) + extname);
```

Comment:
The problem is that the `path` library assumes you are working with a file path (which on Windows uses backslashes) where the `url` library uses URL paths which are always forward slashes. When I tried using the `path.resolve`, it used backslashes as well.